src/utils/types.ts

```diff
@@ -81,39 +81,25 @@ export class RESTAPI extends APIBase {
     protected contentType:string;
     protected acceptType:string;
 
-    get = async (baseurl?:string, contentType?:string, acceptType?:string):Promise<RequestResponseData> => {
+    get = async (baseurl?:string):Promise<RequestResponseData> => {
         let ep:string = baseurl ? baseurl : this.baseurl;
 
-        // Set contentType if exists
```

Comment:
Please don't remove the params, that's not what I asked for. I said don't overwrite the default value. That doesn't mean the params were wrong.